test/helper.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+/**
+ * Removes the line break in the given file contents.
+ */
+exports.removeLineBreak = function (contents) {
+    return contents.toString().replace(/\r?\n|\r/g, '');
+};
```

Comment:
This is for readability of test code. I thought `removeLineBreak(newFile.contents)` is more readable than `newFile.contents.toString().replace(/\r?\n|\r/g, '')` but if you think this is unnecessary, I'm glad to restore the original code (and remove this).